packages/generator-single-spa/src/root-config/templates/src/index.ejs

```diff
@@ -32,10 +32,11 @@
   <script type="systemjs-importmap">
     {
       "imports": {
-        "single-spa": "https://cdn.jsdelivr.net/npm/single-spa@5.5.0/lib/system/single-spa.min.js"
+        "single-spa": "https://cdn.jsdelivr.net/npm/single-spa@5.5.5/lib/system/single-spa.min.js"
       }
     }
   </script>
+  <link rel="preload" href="https://cdn.jsdelivr.net/npm/single-spa@5.5.5/lib/system/single-spa.min.js" as="script">
```

Comment:
Minor perf boost

Reply:
Should this also be added to the layout version of the template? https://github.com/single-spa/create-single-spa/blob/master/packages/generator-single-spa/src/root-config/templates/src/layout/index.ejs

Reply:
It is - https://github.com/single-spa/create-single-spa/pull/165/files#diff-473cf7d0885b1dfb721b1a1067ed11e3R37

Reply:
Ah missed that. LGTM!